.vib/clickhouse/goss/clickhouse.yaml

```diff
@@ -0,0 +1,29 @@
+command:
+  check-clickhouse-server:
+    exec: timeout --preserve-status 5 clickhouse-server 2>&1 || true
+    exit-status: 0
+    stdout:
+      - Ready for connections
+file:
+  /var/lib/clickhouse/tmp:
+    exists: true
+    filetype: symlink
+    linked-to: /opt/bitnami/clickhouse/tmp
+  /opt/bitnami/clickhouse/logs/clickhouse.log:
```

Comment:
I would additionally check that `/opt/bitnami/clickhouse/logs/clickhouse_error.log` exists and is linked to `stderr`